lambdas/requirements.txt

```diff
@@ -9,6 +9,7 @@ certifi==2023.7.22
 cffi==1.15.1
 charset-normalizer==3.2.0
 cryptography==41.0.7
+email-validator==2.1.0.post1
```

Comment:
Is this package still required? Looks like you're using pydantic for email validation

Reply:
It's required as pydantic internally uses this library for email validation:
https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.EmailStr
https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.validate_email
(I use the validate_email instead of EmailStr type as the latter doesn't allow email to be blank)